commands/azureCommands/acr-build-logs.ts

```diff
@@ -0,0 +1,381 @@
+                                    <td class = 'widthControl'>${buildTask}</td>
+                                    <td class ='status widthControl ${log.status}'>${log.status}</td>
+                                    <td class = 'widthControl'>${startTime}</td>
+                                    <td class = 'widthControl'>${finishTime}</td>
+                                    <td class = 'widthControl'>${osType}</td>
+                                    <td><div class = "arrow"></div></td>
+                                </tr>
+                            </table>
+                        </button>
+                        <div class="panel">
+                            <table class="overallTable">
+                                <tr>
+                                    <td colspan="4">Output Images</td>
+                                </tr>
+                                <tr>
+                                    <td>Tag</th>
+                                    <td>Repository</td>
+                                    <td>Registry</td>
+                                    <td>Digest</td>
+                                </tr>
+                                ${imageOutput}
+                            </table>
+                                <div class = 'button-holder'>
+                                    <button id= "log${i}" class="viewLog">Open Logs</button>
+                                </div>
+                        </div>
+                    </div>`
+        });
+    }
+    if (startItem) {
+        panel.webview.postMessage({ 'type': 'endContinued' });
+    } else {
+        panel.webview.postMessage({ 'type': 'end' });
+    }
+}
+/** Create the table in which to push the build logs */
+function getWebviewContent(scriptFile: vscode.Uri, stylesheet: vscode.Uri): string {
+    return `<!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <link rel="stylesheet" type="text/css" href="${stylesheet}">
+        <meta charset="UTF-8">
+        <meta http-equiv="Content-Security-Policy" content="frame-src vscode-resource:; img-src vscode-resource: https:; script-src vscode-resource:; style-src vscode-resource:;">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Logs</title>
+    </head>
+
+    <body>
+        <div id = "header">
+            <table id="headerTable">
+                <th class = 'widthControl'>Build Name </th>
+                <th class = 'widthControl'>BuildTask </th>
+                <th class = 'widthControl'>Status </th>
+                <th class = 'widthControl'>Start Time </th>
+                <th class = 'widthControl'>Finish Time </th>
+                <th class = 'widthControl'>Platform </th>
+                <td></td>
+            </table>
+        </div>
+        <div id = 'core'>
+        </div>
+        <div class = 'loadMoreBtn'>
+            <button id= "loadBtn" class="viewLog">Load More Logs</button>
```

Comment:
Nit: Can you increase the upper padding between the logs and the Load More Logs button? Doubling the current padding might help